client/src/main/java/com/influxdb/client/BucketsQuery.java

```diff
@@ -0,0 +1,112 @@
+package com.influxdb.client;
+
+import javax.annotation.Nullable;
+
+public class BucketsQuery {
+
+    /**
+     * offset  (optional)
+     */
+    @Nullable
+    private Integer offset;
+
+    /**
+     *  (optional, default to 20)
+     */
+    @Nullable
+    private Integer limit;
+
+    /**
+     * The last resource ID from which to seek from (but not including). This is to be used instead of &#x60;offset&#x60;. (optional)
```

Comment:
Fix `JavaDoc` formatting:
```suggestion
     * The last resource ID from which to seek from (but not including). 
     * This is to be used instead of &#x60;offset&#x60;. (optional)
```